drydock/values.yaml

```diff
@@ -0,0 +1,49 @@
+
+# Copyright 2017 The Openstack-Helm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file provides defaults for drydock
+
+replicas:
+  drydock: 3
```

Comment:
If drydock is intended to be stateful, we should consider making it a statefulset instead of restricting the replicas to 1.  Restricting deployments to 1 as a best practice for managing state shouldn't be necessary as stateful sets are first class citizens now.  What are your thoughts?